javascript/javascript2/week1/index.html

```diff
@@ -0,0 +1,126 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Risalamande</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Risalamande</h1>
+    </header>
+
+    <section id="intro">
+      <p>
+        Risalamande is the icing on the cake and the sweet conclusion to a
+        dinner on special days.
+      </p>
+      <p>
+        Every year, I look forward to homemade risalamande with crispy toasted
+        almonds and the vanilla flavor from the speckled vanilla beans.
+      </p>
+      <p>
+        My risalamande turns out especially delicious because the vanilla and
+        sugar simmer with the rice pudding.
+      </p>
+      <p>
+        This creates an amazing aroma and a flavor that is utterly irresistible.
+      </p>
+      <break></break>
```

Comment:
```suggestion
      <br />
```
It's the first time I see a `<break>` tag. I am not sure it exists